server/services/anthropic_client.py

```diff
@@ -0,0 +1,43 @@
+from anthropic import Anthropic
+from dotenv import load_dotenv
+import os
+
+from utilities.prompts.database_schema_representation import *
+
+load_dotenv()
+
+def generate_sql_query_anthrophic(question: str):
+    """
+    Generates an SQL query from a natural language question using the OpenAI API.
+
+    Args:
+        question (str): Natural language question.
+
+    Returns:
+        tuple: A tuple containing the generated SQL query string and the prompt used.
+
+    Raises:
+        RuntimeError: If there is an error with the OpenAI API request.
+    """
+    prompt = (
+        f"### Complete sqlite SQL query only and with no explanation\n"
+        f"### Given the following database schema :\n {code_representation_database_schema}\n"
+        f"### Answer the following: {question}\nSELECT*/"
+    )
```

Comment:
I see that these are duplicated here as well 
https://github.com/Conrad-X/text2SQL/blob/2e1589ed2748c6fb580606c1ea2d511ac5a07fa2/server/services/openai_client.py#L9 
We can fetch it out and use it as a method that we can put in `utilities`